sway-fmt-v2/src/items/item_enum/mod.rs

```diff
@@ -28,12 +28,8 @@ impl Format for ItemEnum {
             write!(formatted_code, "{} ", visibility.span().as_str())?;
         }
         // Add enum token and name
-        write!(
-            formatted_code,
-            "{} {}",
-            self.enum_token.span().as_str(),
-            self.name.as_str()
-        )?;
+        write!(formatted_code, "{} ", self.enum_token.span().as_str(),)?;
```

Comment:
shouldn't this be `write!(formatted_code, "{} ", self.enum_token.span().as_str())?;`

Reply:
I'm guessing rustfmt struggles a little more with `macro_rules!`, as they can basically have arbitrary syntax! I imagine they'll eventually at least add some special cases for these std macros like `println!`, `write!`, etc.

Reply:
Ah, thanks I missed this